api/dao/containerstorage.py

```diff
@@ -23,7 +23,8 @@ def __init__(self, cont_name, use_object_id = False):
     def get_container(self, _id):
         return self._get_el(_id)
 
-    def exec_op(self, action, _id=None, payload=None, query=None, user=None, public=False, projection=None):
+    def exec_op(self, action, _id=None, payload=None, query=None, user=None, 
+                public=False, projection=None, recursive=False):
```

Comment:
This reminds me that we still haven't set a character line limit. I've been casually rolling with 150 by setting my browser to that width, but also going over when convenient.

Let's pick a linter asap and start with some very lax rules.